server/src/instant/db/datalog.clj

```diff
@@ -541,6 +554,7 @@
     :function (let [[func val] (first v-value)]
                 (case func
                   :$not [[:not= :value (value->jsonb val)]]
+                  :$like [[:like [:->> :value :0] val]]
```

Comment:
Can you run an explain on the query that this generates? My expectation is that it's going to have to do a sequential scan over all the triples with `attr_id = :attr-id`. If you wait for my comparison operators PR, I can make it use the string index.